concrete-core/src/specification/engines/ggsw_ciphertext_trivial_encryption.rs

```diff
@@ -0,0 +1,51 @@
+use crate::specification::engines::{engine_error, AbstractEngine};
+use crate::specification::entities::{GgswCiphertextEntity, PlaintextEntity};
+use concrete_commons::parameters::{
+    DecompositionBaseLog, DecompositionLevelCount, GlweSize, PolynomialSize,
+};
+
+engine_error! {
+    GgswCiphertextTrivialEncryptionError for GgswCiphertextTrivialEncryptionEngine @
+}
+
+/// A trait for engines trivially encrypting GGSW ciphertext.
+///
+/// # Semantics
+///
+/// This [pure](super#operation-semantics) operation generates a GGSW ciphertext containing the
+/// trivial encryption of the `input` plaintext vector with the requested `glwe_size`.
+///
+/// # Formal Definition
+///
+/// A trivial encryption uses a zero mask and no noise.
+/// It is absolutely not secure, as the body contains a direct copy of the plaintext
```

Comment:
I would maybe give an example to explain why this is important to have these insecure encryptions.

Reply:
Do you have a sentence in mind for that ? Also, we could add it to every other trivial encryption entry points. Maybe we could leave that to a next PR ? (chaging everything in the same PR instead of doing it here, because the PR is already huge ^^)